File: mobile/lib/features/laundry_service/domain/entities/service.dart
import 'package:equatable/equatable.dart';

class Service extends Equatable {
  final int id;
  final String name;
  final double price;
  final String unit;

  const Service({
    required this.id,
    required this.name,
    required this.price,
    required this.unit,
  });

  @override
  List<Object?> get props => [id, name, price, unit];
}

File: backend/app/Models/Service.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class Service extends Model
{
    use SoftDeletes;
    protected $primaryKey = 'idservices';

    protected $fillable = [
        'categories_idcategories',
        'name',
        'price',
        'unit',
        'estimate_days',
        'is_active'
    ];

    // Relasi: Satu Layanan dimiliki oleh satu Kategori (Belongs To)
    public function category()
    {
        return $this->belongsTo(Category::class, 'categories_idcategories', 'idcategories');
    }
}